raytracing/tests/testsImagingPath.py

```diff
@@ -354,6 +354,25 @@ def testChiefRayInfiniteFOVNoY(self):
         with self.assertRaises(ValueError):
             path.chiefRay()
 
+    def testBugNegativeFNumber(self):
+        # Assuming:
+        path = ImagingPath()
+        path.fanAngle = 0.2094
```

Comment:
Is this necessary?

Reply:
Test passes without setting fan angle

Reply:
Test doesn't work with just 1 divergent lens, it returns a good test. I don't know what's the minimal system to generate a negative Number, but that combinaison of lenses seem to work.

Reply:
But "maybe" the effectiveFocal length is not negative anymore (I know that is not the case, but we must stay rigourous).  The test should include a check that the effectiveFocalLength is indeed negative but the f-number is positive.

Reply:
no I tested it, It produces a negative effective focal length. Yes adding a test could be done.

Reply:
Oh ok I get it now ahah, yes @Valeriouus good test for you! :-)

Reply:
seems ready to merge now.